ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -586,6 +590,7 @@ export const DataAccessRequirements = fp.flow(withProfileErrorModal)((spinnerPro
         <div style={styles.pleaseComplete}>
           Please complete the necessary steps to gain access to the <AoU/> datasets.
         </div>
+        <Iframe frameBorder={10} url={rasLogoutUrl}/>
```

Comment:
Should this always be rendered, or only when the user is logged in to RAS?

Reply:
ah, that is added for my testing purpose, reverted